_ja/overviews/scala3-book/scala-for-python-devs.md

```diff
@@ -0,0 +1,1383 @@
+---
+title: Scala for Python Developers
+type: chapter
+description: This page is for Python developers who are interested in learning about Scala 3.
+languages: [zh-cn, ja]
+num: 76
+previous-page: scala-for-javascript-devs
+next-page: where-next
+---
+
+{% include_relative scala4x.css %}
```

Comment:
The CI fails because on this line:
```
Could not locate the included file 'scala4x.css' in any of ["/home/runner/work/docs.scala-lang/docs.scala-lang/_ja/overviews/scala3-book"]
```

Could you copy this file to `_ja/overviews/scala3-book/scala4x.css`? I am not sure that's the best solution but that's what we did already for `zh-cn`. See [_zh-cn/overviews/scala3-book/scala4x.css](https://github.com/scala/docs.scala-lang/blob/main/_zh-cn/overviews/scala3-book/scala4x.css).